style.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+
+.allinone {
+  position: absolute;
+  background: #fff;
+}
+
+.font {
+  font-family: Poppins, sans-serif;
+}
+
+.navcontainer {
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+  padding: 0%;
+  margin-top: 0;
+}
+
+.navbar-logo {
+  width: 76px;
+  height: 20px;
+  margin-left: 24px;
+  margin-top: 60px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  list-style-type: none;
+  text-decoration: none;
+}
+
+.navbar-toggle {
+  margin-right: 24px;
+  margin-left: 0;
+  margin-top: 60px;
+  list-style-type: none;
+}
+
+.container {
+  background-image: url(img/header-shapes\ mobile.png);
+  width: 375px;
+  height: 604px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.headerone {
+  height: 146px;
+  width: 320px;
+  margin-left: 24px;
+  margin-top: 104px;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  font-style: normal;
+  letter-spacing: 0;
+  text-align: left;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.text {
+  height: 146px;
+  width: 327px;
+  margin-left: 24px;
+  margin-top: -14px;
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  color: #344563;
+}
+
+.connect {
+  width: 120px;
+  height: 24px;
+  margin-left: 24px;
+  margin-top: -4px;
+  letter-spacing: 0;
+  text-align: left;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.socialmedia {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  height: 20px;
+  width: 170px;
+  padding: 0%;
+  margin-left: 24px;
+  top: 470px;
```

Comment:
- [ ] I noticed that while you tried to implement the changes requested by the first reviewer, the UI design of your project was affected. There are white patches which shouldn't be seen. Kindly find the screenshot of what your application looks like on mobile view

![image](https://user-images.githubusercontent.com/77106962/219161305-e7f7a645-a00a-4f41-8c91-d1dd96d9a27a.png)

#### Hint; You need to make your design identical to that of the Figma template that you chose. If you are developing your design, kindly design on the developer's pane of `iPhone X`. This is because it is the standard view for most mobile applications. Your design should look like the screenshot below on the mobile screen: 👍 😎 

![image](https://user-images.githubusercontent.com/77106962/219165269-fbdf4cf8-5201-476b-a543-8a5ec4563194.png)

Reply:
Done